Tests/MongoSwiftSyncTests/ChangeStreamTests.swift

```diff
@@ -162,7 +161,7 @@ internal struct ChangeStreamTest: Decodable {
     internal func run(globalClient: MongoClient, database: String, collection: String) throws {
         let client = try MongoClient.makeTestClient(options: ClientOptions(commandMonitoring: true))
 
-        let center = client.notificationCenter
+        let center = NotificationCenter.default
```

Comment:
this was to get rid of the `@testable`. since we aren't ever passing in a notification center when we create the client, it will always be the default. also, this API is going away soon anyway.